src/utils/moterUtil.ts

```diff
@@ -13,19 +13,19 @@ export enum MoteType {
 }
 
 export const moteStatusTekst = (mote: MoteDTO): string => {
-  const prefix = "Planlegger:";
+  const prefix = "Planlegger";
   switch (mote.status) {
     case MoteStatus.OPPRETTET: {
-      return `${prefix} Forslag sendt`;
+      return `${prefix} forslag sendt`;
     }
     case MoteStatus.BEKREFTET: {
-      return `${prefix} Bekreftelse sendt`;
+      return `${prefix} bekreftelse sendt`;
     }
     case MoteStatus.FLERE_TIDSPUNKT: {
-      return `${prefix} Endring sendt`;
+      return `${prefix} endring sendt`;
     }
     case MoteStatus.AVBRUTT: {
-      return `${prefix} Avbrutt`;
+      return `${prefix} avbrutt`;
```

Comment:
Blir ikke dette dårlig norsk? Hva er f.eks "Planlegger mellomrom forslag".

Reply:
Enig i at det ser litt rart ut. Jeg hører med Espen om vi kan finne på noe bedre.

Reply:
Fikset nå. Gikk for:
Planlegger: Forslag
Planlegger: Endring
Planlegger: Bekreftet
Planlegger: Avbrutt